database/migrations/mysql/1583505136_add_vsphere_pke_clusters_table.up.sql

```diff
@@ -0,0 +1,11 @@
+CREATE TABLE `vsphere_pke_clusters` (
+  `id` int(10) unsigned NOT NULL AUTO_INCREMENT,
+  `created_at` timestamp NULL DEFAULT NULL,
+  `updated_at` timestamp NULL DEFAULT NULL,
+  `deleted_at` timestamp NULL DEFAULT NULL,
```

Comment:
Currently only the `clusters` table is soft-deleteable, ideally it shouldn't be either. I'd like to see this gone for now and reintroduced if/when there is a decision about making other tables soft-deletable as well.

Reply:
Still soft deleteable.